### PR TITLE
Added additional temporary e-mail domain names

### PR DESCRIPTION
**Pull request by brtdv**
@martenson I've added a few additional temporary e-mail domain names to the list, cheers.

### CODE REVIEW

File: disposable_email_blocklist.conf [hunk 1]
```diff
@@ -799,6 +809,7 @@ ecallheandi.com
 ecolo-online.fr
 edgex.ru
 edinburgh-airporthotels.com
+edu.sg
```
Comment (di) [inline disposable_email_blocklist.conf]: This is not a domain, it's a public suffix, which is causing CI to fail here.

**Comment by di** [comment]
Hi @brtdv, thanks for the contribution. Can you provide a source for these domains? E.g., where can I go to get a `@antichef.com` temporary email address?

Thanks!

**Comment by brtdv** [comment]
@di Good question, I've tried looking for the source by looking back in the commit history of my web app, which contains a similar list of domain names that are blocked.

I was looking to update my list in my webapp, since the latest commit to my file was from june '18. That's how I found [martenson's repo](https://github.com/martenson/disposable-email-domains), which I then used to update the list of 'blocked' e-mail domains in my web app. I thought I'd contribute back by adding all missing domains that were in my list, which weren't in the repo...

So that's the reason behind my commit... Unfortunately I'm not able to provide a real 'source'. You can. however, find the domain name on [check-mail.org](https://check-mail.org/domain/antichef.com/) and verify that this is indeed a temporary e-mail domain name, that can be created by using [spamgourmet.com](https://www.spamgourmet.com/index.pl).

File: disposable_email_blocklist.conf (hunk 1 above)
Comment (brtdv) [reply disposable_email_blocklist.conf]: You are probably correct, this [does not seem like a disposable e-mail address](https://check-mail.org/domain/edu.sg/). My bad, should I add an extra commit removing this item?

**Comment by DanielAbsi** [comment]
Hi I found new temporary emai to be added: 
mailnd7.com
qortu.com
psk3n.com
tashjw.com
ifavorsprt.com
andyes.net

File: disposable_email_blocklist.conf (hunk 1 above)
Comment (di) [reply disposable_email_blocklist.conf:812]: Yes, please remove it. Thanks!

**Comment by di** [comment]
Thanks @brtdv. Without a source it's hard to verify these, e.g. https://check-mail.org/domain/10minutemail.it/ says that this is not a disposable domain (and doesn't even have an MX record).

I'd be OK with merging any domains that a) have an MX record, and b) that MX record points to a domain already on the list.

**Comment by martenson** [comment]
Since this PR has stalled I am closing it. Please feel free to reopen when you can provide source for added domains.

Thanks for contributing.

**Pull request closed without merging**